ui-backend/catalog-ui-search-api/src/main/java/org/codice/ddf/catalog/ui/events/EventTypeDeserializer.java

```diff
@@ -0,0 +1,26 @@
+/**
+ * Copyright (c) Codice Foundation
+ *
+ * <p>This is free software: you can redistribute it and/or modify it under the terms of the GNU
+ * Lesser General Public License as published by the Free Software Foundation, either version 3 of
+ * the License, or any later version.
+ *
+ * <p>This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY;
+ * without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Lesser General Public License for more details. A copy of the GNU Lesser General Public
+ * License is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ */
+package org.codice.ddf.catalog.ui.events;
+
+import com.google.gson.*;
```

Comment:
❗ `*` Imports should be removed in favor of explicit exports. I know sometimes your IDE will sneak up and do this unawares, there should be a setting for it in intellij in case you don't have it set to stop it.
![image](https://user-images.githubusercontent.com/9013416/88981469-e0f1af80-d27a-11ea-9fae-3d0c1df45bc9.png)

Reply:
thank you!